### DISCUSSION TITLE
Particles not loading on mobile devices - getting error

### DISCUSSION THREAD

**Discussion by stylessard** [matteobruni/tsparticles#2427] [Q&A] [answered]
Hi everyone! Firstly, I should note that I'm using the React version of tsParticles. Wasn't sure if this error is specific to React so I posted here. If you'd like me to switch the convo to over there, just let me know!

Still playing around with tsParticles. I've got everything up and running properly on desktop, but when I switch to an iPad or iPhone, the particles fail to load. In the console, I get this error - 

>Unhandled Promise Rejection: TypeError: element.addEventListener is not a function. (In 'element.addEventListener(event, handler, addOptions)', 'element.addEventListener' is undefined)

The error traces to this function - 
```

function manageListener(element, event, handler, add, options) {
  if (add) {
    let addOptions = {
      passive: true
    };

    if (typeof options === "boolean") {
      addOptions.capture = options;
    } else if (options !== undefined) {
      addOptions = options;
    }

    // this line below specifically 
    element.addEventListener(event, handler, addOptions);
  } else {
    const removeOptions = options;
    element.removeEventListener(event, handler, removeOptions);
  }
}
```

More than likely an easy fix, but I've been unsuccessful in figuring it out. Tried removing options one by one as a shot in the dark fix, but no bueno. Any suggestions?

Thank you!

**Comment by matteobruni** [comment, marked as answer]
Hi @stylessard, can you open an issue about it and give ne more details about errore message, device version, tsParticles version or the configuration used
